osu.Game/Online/Multiplayer/Room.cs

```diff
@@ -103,38 +103,48 @@ private int duration
         [JsonIgnore]
         public readonly Bindable<int> Position = new Bindable<int>(-1);
 
-        public void CopyFrom(Room other)
+        /// <summary>
+        /// Copies the properties from another <see cref="Room"/> to this room.
+        /// </summary>
+        /// <param name="other">The room to copy</param>
+        /// <param name="duplicate">Whether the copy should exclude information unique to a specific room (i.e. when duplicating a room)</param>
+        public void CopyFrom(Room other, bool duplicate = false)
```

Comment:
I really dislike this `duplicate` param. maybe the actual "duplicating" functionality should be an entirely separate method? something like `CreateCopyOf(Room)`?